pkg/controllers/dynakube/connectioninfo/reconciler.go

```diff
@@ -36,7 +37,19 @@ func NewReconciler(clt client.Client, apiReader client.Reader, scheme *runtime.S
 	}
 }
 
+func (r Reconciler) updateDynakubeStatus(ctx context.Context) error {
+	r.dynakube.Status.UpdatedTimestamp = metav1.Now()
+	err := r.client.Status().Update(ctx, r.dynakube)
+	if err != nil {
+		log.Error(err, "could not update dynakube status", "name", r.dynakube.Name)
+		return err
```

Comment:
pls don't log the `err` if you are returning it because you will cause a double log:

you can either:
- do a `return errors.WithMessage(err, <message>)`
- use `log.Info` without using the `err`


[I update the style-guide PR with an example](https://github.com/Dynatrace/dynatrace-operator/pull/2383/files#diff-7c8612f5428f9d88cdd38edbaa31cd4d455a4d9736a3122efb3132444c180fc8R126-R149)

Reply:
fixed